victor/src/layout/boxes/generation.rs

```diff
@@ -198,13 +180,11 @@ impl<'a> BlockContainerBuilder<'a> {
                 .current_parent_style()
                 .expect("found a text node without a parent")
                 .clone();
-            self.ongoing_inline_formatting_context
-                .text_runs
-                .push(IntermediateTextRun {
-                    parent_style,
-                    node: descendant,
-                });
-            self.push_inline_level_box(InlineLevelBox::TextRun(run_id));
+            let text = match &self.context.document[descendant].data {
```

Comment:
Isn't that the argument `contents` already?

Reply:
Yes, it’s removed in the next commit.